src/EasyAdminBundle/Controller/UserControllerPreventingFromSelfDelete.php

```diff
@@ -1,14 +1,14 @@
 <?php
 
-namespace AppBundle\Controller\Admin;
+namespace EasyAdminBundle\Controller;
 
 use Doctrine\DBAL\Exception\ForeignKeyConstraintViolationException;
 use JavierEguiluz\Bundle\EasyAdminBundle\Controller\AdminController as BaseAdminController;
 use JavierEguiluz\Bundle\EasyAdminBundle\Event\EasyAdminEvents;
 use JavierEguiluz\Bundle\EasyAdminBundle\Exception\EntityRemoveException;
 use Symfony\Component\HttpFoundation\RedirectResponse;
 
-class UserController extends BaseAdminController
+class UserControllerPreventingFromSelfDelete extends BaseAdminController
```

Comment:
I know I told you to do so, but now another idea appeared in my mind.

Let your class extend `AdminController` instead of `BaseAdminController` which is unnecessary alias right now. Then the class name should be `AdminControllerPreventingFromSelfDelete`. 

Such approach makes sense, is easy to understand and follow bundle inheritance rules.